### DISCUSSION TITLE
Can't get gridtracker to connect to cloudlog

### DISCUSSION THREAD

**Discussion by lyonrt** [magicbug/Cloudlog#1068] [Q&A]
What is the url path to paste into gridtracker?  I put in different variations with the api key and it keeps saying invalid.
I am using https://www.ko4nek.com/cloudlog/api

**Comment by ofadam** [comment]
It doesn't look like your domain resolves correctly?

**Reply by ofadam** [reply]
But it should be /cloudlog/index.php/api/qso (with index.php optional depending on your config)
